packages/react-server-cli/package.json

```diff
@@ -44,8 +46,9 @@
     "sass-loader": "^4.0.0",
     "style-loader": "~0.12.3",
     "webpack": "^1.13.1",
-    "webpack-dev-server": "~1.13.0",
-    "webpack-stats-plugin": "^0.1.3",
+    "webpack-dev-middleware": "^1.9.0",
+    "webpack-hot-middleware": "^2.13.2",
+    "webpack-stats-plugin": "^0.1.4",
```

Comment:
It looks like this module is not actually required. The generated JSON file is never read. Or am I wrong?